fellowship.js

```diff
@@ -17,32 +17,106 @@ var fellowship = {
     "The Shire", "Rivendell", "Mordor"
   ],
   makeMiddleEarth: function(lands) {
-    console.log(lands)
+    // console.log(lands);
+    var newSection = document.createElement('section');
+    newSection.id='middle-earth';
+    for (var i in fellowship.lands){
```

Comment:
check out http://stackoverflow.com/questions/500504/why-is-using-for-in-with-array-iteration-such-a-bad-idea to see why to avoid `for .. in` w/ javascript arrays